From 4a3eaf1780310d241e9afa7f39b9f4a70d5cf445 Mon Sep 17 00:00:00 2001
From: root <root@ubuntu.com>
Date: Sun, 10 May 2015 13:34:48 +0800
Subject: [PATCH] 1 a.c

---
 README | 0
 a.c    | 1 +
 2 files changed, 1 insertion(+)
 create mode 100644 README
 create mode 100644 a.c

diff --git a/README b/README
new file mode 100644
index 0000000..e69de29
diff --git a/a.c b/a.c
new file mode 100644
index 0000000..2b93201
--- /dev/null
+++ b/a.c
@@ -0,0 +1 @@
+aaaaaaaaa
-- 
1.9.1

